charts/posthog/templates/clickhouse-operator/deployment.yaml

```diff
@@ -38,8 +47,8 @@ spec:
             name: etc-clickhouse-operator-usersd-files
       containers:
         - name: clickhouse-operator
-          image: altinity/clickhouse-operator:latest 
-          imagePullPolicy: IfNotPresent
+          image: altinity/clickhouse-operator:latest
```

Comment:
off topic

should this be pinned to a specific version?

Reply:
I think it's up to upstream to decide unfortunately